src/stylesheets/utility/_tag-list.scss

```diff
@@ -1,20 +1,173 @@
 .tag-list {
   list-style-type: none;
   padding: 0;
+  position: relative;
+
+  @media (screen(lg)) {
+    height: rem(800);
+    margin-top: rem(-800);
+  }
 }
 
 .tag-list__item {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
   align-items: center;
   background-color: color(white);
   border-radius: rem(4);
-  margin: 0 0 rem(30) 0;
-  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto rem(30) auto;
+  max-width: rem(500);
   padding: 0 rem(12) rem(24) rem(12);
+  position: relative;
 
   a {
     color: color(coral-pink);
   }
+
+  @media (screen(lg)) {
+    box-sizing: border-box;
+    border-radius: rem(100) rem(10) rem(10) rem(100);
+    padding: rem(32) rem(22) rem(32) rem(82);
+
+    .-orange,
+    .sm-sans-serif-text {
+        text-align: left;
+      }
+
+    &.-alt {
+      border-radius: rem(10) rem(100) rem(100) rem(10);
+      padding: rem(32) rem(82) rem(32) rem(22);
+
+      .-orange,
+      .sm-sans-serif-text {
+        text-align: right;
+      }
+    }
+  }
+}
+
+.factors-section {
+
+  @media (screen(lg)) {
+    .tag-list__item {
+      max-width: rem(360);
+      position: relative;
+     
+      &:nth-child(1) {
+        top: 13%;
+        left: 22%;
+
+        .-gear {
+          top: 29%;
+          left: -11%;
+        }
+      }
+
+      &:nth-child(2) {
+        top: 6%;
+        left: -16%;
+
+        .-heart {
+          top: 26%;
+          left: 89%;
+        }
+      }
+
+      &:nth-child(3) {
+        top: 3%;
+        left: 25%;
+
+        .-dollar {
+          top: 29%;
+          left: -11%;
+        }
+      }
+
+      &:nth-child(4) {
+        top: -13%;
+        left: -30%;
+
+        .-tally {
+          top: 27%;
+          left: 89%;
```

Comment:
Instead of positioning the "tally" button (and heart button) with top and left I think you could just target them both with the `.-alt` modifier, e.g. `.tag-list__item.-alt .tag-list__btn { /* position it on the right/center */}`.

For right center, you can also try:
```
margin: auto;
top: 0;
bottom: 0;
```

Then `transform: translateX(50%)` to move it exactly halfway off the edge of the tag. Let me know if you want help with this too.